### ISSUE TITLE
Questions regarding Intent Recognition with BERT using Keras and TensorFlow 2

### ISSUE THREAD

**Issue by bastia0321** [curiousily/Deep-Learning-For-Hackers#1]
Thanks for sharing your work. That is really helpful. However in the tutorial "Intent Recognition with BERT using Keras and TensorFlow 2", why the valid loss/acc do not change by epoches? The training loss/acc are improved but the model seems to have good performance on the valid dataset before any finetuning.
